components/library/yaml-cpp/yaml-cpp.p5m

```diff
@@ -107,24 +60,12 @@ file path=usr/include/yaml-cpp/parser.h
 file path=usr/include/yaml-cpp/stlemitter.h
 file path=usr/include/yaml-cpp/traits.h
 file path=usr/include/yaml-cpp/yaml.h
-file path=usr/lib/$(MACH64)/cmake/GTest/GTestConfig.cmake
-file path=usr/lib/$(MACH64)/cmake/GTest/GTestConfigVersion.cmake
-file path=usr/lib/$(MACH64)/cmake/GTest/GTestTargets-release.cmake
-file path=usr/lib/$(MACH64)/cmake/GTest/GTestTargets.cmake
 file path=usr/lib/$(MACH64)/cmake/yaml-cpp/yaml-cpp-config-version.cmake
 file path=usr/lib/$(MACH64)/cmake/yaml-cpp/yaml-cpp-config.cmake
 file path=usr/lib/$(MACH64)/cmake/yaml-cpp/yaml-cpp-targets-release.cmake
 file path=usr/lib/$(MACH64)/cmake/yaml-cpp/yaml-cpp-targets.cmake
-file path=usr/lib/$(MACH64)/libgmock.so
-file path=usr/lib/$(MACH64)/libgmock_main.so
-file path=usr/lib/$(MACH64)/libgtest.so
-file path=usr/lib/$(MACH64)/libgtest_main.so
-link path=usr/lib/$(MACH64)/libyaml-cpp.so target=libyaml-cpp.so.0.7
-file path=usr/lib/$(MACH64)/libyaml-cpp.so.$(COMPONENT_VERSION)
-link path=usr/lib/$(MACH64)/libyaml-cpp.so.0.7 \
-    target=libyaml-cpp.so.$(COMPONENT_VERSION)
-file path=usr/lib/$(MACH64)/pkgconfig/gmock.pc
-file path=usr/lib/$(MACH64)/pkgconfig/gmock_main.pc
-file path=usr/lib/$(MACH64)/pkgconfig/gtest.pc
-file path=usr/lib/$(MACH64)/pkgconfig/gtest_main.pc
+link path=usr/lib/$(MACH64)/libyaml-cpp.so target=libyaml-cpp.so.0.8
```

Comment:
Since the library version changed it will probably cause that its consumers will stop to work (`qtcreator` and `opencolorio`).

Reply:
I think the best way how to solve this is to do something similar here as I did with gspell in #19260.